src/component/feature/FeatureToggleList/FeatureToggleListItem/FeatureToggleListItem.jsx

```diff
@@ -126,7 +144,10 @@ const FeatureToggleListItem = ({
                     <ConditionallyRender
                         condition={hasAccess(UPDATE_FEATURE, project)}
                         show={
-                            <IconButton onClick={() => revive(feature.name)}>
+                            <IconButton
```

Comment:
We should use PermissionIconButton here, this action should only be allowed if you have access to UPDATE_FEATURE.